docs/rhoas/iac_tools_rhoas/README.adoc

```diff
@@ -0,0 +1,602 @@
+
+[source, subs="+quotes"]
+----
+kafka_id: __<kafka_id>__
+----
+====
++
+. Click on the copy symbol to copy the contents of the example playbook.
+. Open the `rhoas_test.yml` file in your IDE and paste the example into it.
+. Configure the `create_kafka_` module to provision a new Kafka instance. The only required fields for this module are the `name`, `billing_model` and `cloud provider` fields. You can specify configuration options such as `billing_cloud_account_id` and the `openshift_offline_token`. All other information for the instance is provided by the Kafka APIs. The default contents of this module in the OpenShift Application Services example playbook are shown below:
++
+.Example `create_kafka` module
+[source,yaml]
+----
+- name: Create kafka
+    rhoas.rhoas.create_kafka:
+      name: "kafka-name"
+      instance_type: "x1"
+      billing_model: "standard"
+      cloud_provider: "aws"
+      region: "us-east-1"
+      plan: "developer.x1"
+      billing_cloud_account_id: "123456789"
+      openshift_offline_token: "OPENSHIFT_OFFLINE_TOKEN"
+    register:
+      kafka_req_resp
+----
++
+When you run the `create_kafka_` module, Ansible saves the output of that command in a variable in the `register` field. In the preceding example, Ansible saves the created Kafka instance as `_kafka_req_resp_`.
+
+. Configure the `create_service_account` module to create a service account. The only required fields are the `name` and `short description` fields. Ansible populates the generated service account credentials in the `client_id` and `client_secret` fields once it creates the service account.
++
+.Example `create_service_account` module
+[source,yaml]
+----
+- name: Create Service Account
+    create_service_account:
+      name: "service-account-name"
+      description: "This is a description of the service account"
+      openshift_offline_token: "OPENSHIFT_OFFLINE_TOKEN"
+    register:
+      srvce_acc_resp_obj
+----
+
+. Configure the `create_kafka_acl_binding` module to create an ACL for the service account with some default values, and bind that ACL to the Kafka instance and service account. You can get the Kafka ID from the `_kafka_req_resp_` variable or enter it in the kafka_id field. You can get the service account ID from the `_srvce_acc_resp_obj_` variable. The following list is a description of each field that must have a value in an ACL binding module.
+** `resource_name`: the name of resource you want access to. This example uses the name that is passed when creating the topic.
+** `resource_type`: The type of resource you want access to. This example uses *Topic*.
+** `pattern_type`: The type of pattern of the ACL.
+** `operation_type`: The type of operation that is allowed for the given user on this module.
+** `permission_type`: Whether permission is given or taken away.
```

Comment:
I'd use a definition list here:

```
** `resource_name`: the name of resource you want access to. This example uses the name that is passed when creating the topic.
`resource_type`:: The type of resource you want access to. This example uses `Topic`.
`pattern_type`:: The type of pattern of the ACL.
`operation_type`:: The type of operation that is allowed for the given user on this module.
`permission_type`:: Whether permission is given or taken away.
```

Reply:
You might want to say (in the descriptions above) what some of the possible values are for each of the those fields. For example, I would have no idea what "the type of pattern of the ACL" means (`pattern_type`), or what some of the possible values are, apart from the `PREFIXED` value shown in the example.